crates/ibc/src/core/ics04_channel/msgs/acknowledgement.rs

```diff
@@ -47,6 +47,19 @@ impl AsRef<[u8]> for Acknowledgement {
     }
 }
 
+impl From<&[u8]> for Acknowledgement {
+    fn from(bytes: &[u8]) -> Self {
+        Self(bytes.to_vec())
+    }
+}
+
+#[cfg(any(test, feature = "mocks"))]
+impl Default for Acknowledgement {
+    fn default() -> Self {
```

Comment:
Though this default method is under the `Mocks` feature, I was wondering if we could use `from()` instead for tests. 
My understanding is that default usually refers to a convenient, valid value that results in a safe behaviour, even if the caller doesn't supply a field, so the system calls that from default. WDYT?

Reply:
You're right, it wasn't really a default value. [c5415e6](https://github.com/cosmos/ibc-rs/pull/364/commits/c5415e637fb112382d25d000075de8a7ec300a34)

Reply:
🙏🏻